tasks/copy.js

```diff
@@ -428,6 +428,25 @@ gulp.task('copy:dist:component:mock', function () {
   return copyFiles(from, to);
 });
 
+gulp.task('copy:dist:component:data', function () {
+  const from = path.join(
+    config.global.cwd,
+    config.global.src,
+    config.global.components,
+    '**',
+    'data',
```

Comment:
how do we decide if a folder gets an underscore prefix or not?

Reply:
If you refer to a pattern that forbids building this resource...we don't have one. So _ files/folders will also be built.

Reply:
folders with _ are folders which are necessary for the dev environment/tests/styleguide but NOT used in live environments.

Reply:
@timomayer true, but we're not handling them here right? @dannystey maybe you could open a new ticket for that, so we handle that for all folders :)